Dockerfile.mrvlVSP.rhel

```diff
@@ -0,0 +1,31 @@
+FROM registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.22-openshift-4.17 AS builder
```

Comment:
4.18

Reply:
Currently TOT (Dockerfile.daemon.rhel) shows 4.17, can we move along with TOT?

Reply:
This is ok for now. We can adjust later.